ecl/eclcc/eclcc.cpp

```diff
@@ -257,6 +257,9 @@ class EclCC
         if (isEmptyString(defaultGitPrefix))
             defaultGitPrefix = "https://github.com/";
         optDefaultGitPrefix.set(defaultGitPrefix);
+        const char * gitUser = getenv("HPCC_GIT_USERNAME");
+        if (!isEmptyString(gitUser))
+            optGitUser.set(gitUser);
```

Comment:
should HPCC_GIT_PASSPATH environment var be processed here too?

Reply:
no.  Really this is purely for backward compatibility to ensure that existing configurations still work.  It could be deleted in a later version.